.pre-commit-config.yaml

```diff
@@ -1,17 +1,27 @@
 repos:
--   repo: https://github.com/pycqa/isort
-    rev: 5.10.1
-    hooks:
-    - id: isort
--   repo: https://github.com/psf/black
-    rev: 22.1.0
-    hooks:
-    - id: black
-      pass_filenames: true
```

Comment:
`pass_filenames` is `true` by default so I see no benefit of having it here https://pre-commit.com/#creating-new-hooks